key_management/src/keystore.rs

```diff
@@ -63,7 +63,7 @@ pub mod json {
     use serde::{de, Deserialize, Deserializer, Serialize, Serializer};
 
     /// Wrapper for serializing and deserializing a SignedMessage from JSON.
-    #[derive(Clone, Deserialize, Serialize)]
+    #[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
It's hard to tell is this trait still needed?

Reply:
Not anymore, good catch. Was used for debugging